common/k8s/src/event_source.rs

```diff
@@ -240,7 +240,7 @@ impl K8sEventStream {
         pod_label: String,
     ) -> Result<Self, K8sError> {
         Ok(Self {
-            client: Client::new(config.try_into()?),
```

Comment:
All of the changes in common/k8s are just updating to the new kube-rs api. There are no changes here